src/main/resources/com/ncc/aif/aida_ontology.shacl

```diff
@@ -133,10 +136,8 @@ aida:EventShape
     # events must have unique identifiers
     sh:nodeKind sh:IRI ;
 
-    # may have zero or more string labels for user consumption
-    sh:property [
-     sh:path skos:prefLabel ;
-     sh:datatype xsd:string ];
+    # may provide an informative justification
+    sh:property aida:InformativeJustificationPropertyShape ;
```

Comment:
Does the informative justification obviate the need for the prefLabel here and in RelationShape, or is it being removed for its own reason?  And if the latter, should it also be removed from EventArgumentShape and SentimentShape?  I do seem to recall you saying something about not supporting SKOS.

Reply:
It was removed because AIF doesn't use SKOS. I've been removing them as I come across them, but I'm OK to remove them all with this branch if that is preferable. Let me know what you'd like done.

Reply:
Sure, let's nuke 'em (and the prefix).  There are only two more.

Reply:
Removed. After merging with master, I was only able to find one more reference